icu4c/source/test/intltest/restsnew.cpp

```diff
@@ -1395,13 +1396,22 @@ void NewResourceBundleTest::TestFilter() {
     }
 }
 
+/*
+ * The following test for ICU-20706 has infinte loops on certain inputs for
+ * locales and calendars.  In order to unblock the build (ICU-21055), those
+ * specific values are temporarily removed.
+ * The issue of the infinite loops and its blocking dependencies were captured
+ * in ICU-21080.
+ */
+
 void NewResourceBundleTest::TestIntervalAliasFallbacks() {
     const char* locales[] = {
         // Thee will not cause infinity loop
         "en",
         "ja",
 
         // These will cause infinity loop
+        #if 0
```

Comment:
Please remove the indents from the four lines with `#`.

Reply:
Done.